### ISSUE TITLE
AdGuard DNS scheduling feature

### ISSUE THREAD

**Issue by PavelParkhomenko** [AdguardTeam/AdGuardDNS#817]
### Issue Details

Currently, internet connections can only be blocked by manually enabling a user rule, which can be inconvenient if this needs to be done daily.

### Proposed solution

Introduce the ability to set rules or schedules that automatically allow or block internet connections based on specified timeframes. For example, users could schedule internet access to be allowed only during work hours, such as 08:00-16:00

**Comment by emeritaacuity0u** [comment]
Possible duplicate of #574.

**Comment by Chinaski1** [comment]
Closed as duplicate

**Comment by ihurin** [comment]
As a possible improvement of this feature, other customers ask to make user rules on schedule as well. So that access to specific websites is restricted on time. Or maybe this can be (in)activity period for a specific server.

**Comment by gitfirewall** [comment]
I strongly request this feature!
